.github/workflows/ci.yaml

```diff
@@ -0,0 +1,47 @@
+name: "CI"
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: "0 7 * * *"
+
```

Comment:
Can we put in a concurrency call here? see: https://github.com/OpenFreeEnergy/Lomap/blob/b31e6cde3f8e4117e75fd592ac925c17b77cae71/.github/workflows/CI.yaml#L14